packages/langchain-graph-retriever/tests/test_traversal_mmr.py

```diff
@@ -1,12 +1,10 @@
+from graph_retriever.strategies import Mmr
```

Comment:
This and `test_traversal_eager.py` should be significantly trimmed down in future PRs. They should just verify that the strategies can be manipulated, arguments passed, etc.